cookieconsent.php

```diff
@@ -58,12 +58,30 @@ public function onTwigTemplatePaths()
     public function onTwigSiteVariables()
     {
         $twig = $this->grav['twig'];
+        $assets = $this->grav['assets'];
+        $cookieConsentOptions = $this->config['plugins.cookieconsent'];
 
-        $config = $this->config->toArray();
+        // Configuration
+        $builtInCss = $cookieConsentOptions['built_in_css'];
+        $builtInJs = $cookieConsentOptions['built_in_js'];
+        $loadJsOptionsBottom = $cookieConsentOptions['load_js_options_bottom'];
 
-        $this->grav['assets']->addCss("//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.css");
-        $this->grav['assets']->addJs("//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.js");
+        // Load cookie consent CSS if enabled
+        if (!$this->isAdmin() && $builtInCss) {
```

Comment:
the `isAdmin()` in the if-condition is not required because it is already excluded the `onPluginsInitialized` method
```
    /**
     * Initialize the plugin
     */
    public function onPluginsInitialized()
    {
        // Don't proceed if we are in the admin plugin
        if ($this->isAdmin()) {
            return;
        }
        // Enable the main event we are interested in
        $this->enable([
            'onTwigTemplatePaths' => ['onTwigTemplatePaths', 0],
            'onTwigSiteVariables' => ['onTwigSiteVariables', 0],
        ]);
    }
```

Reply:
That's true! We can't remove this condition.